source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -799,6 +804,11 @@ def __init__(self):
     def fill_journeys(self, request_type, api_request, instance):
         logger = logging.getLogger(__name__)
 
+        # early return
+        if api_request['timeframe_duration'] == api_request['min_nb_journeys'] == 0:
```

Comment:
I understand that we could do an early return if `timeframe_duration` is null, but having a `min_nb_jouney=0` means that we could potentially have results still. 

Having an early return with `api_request['max_nb_journeys'] == 0` or `api_request['count'] == 0` would make more sense to me.

Reply:
In fact, the *early return* concern only the timeframe_duration.
it's better like that:

```python
# early return
if (api_request['timeframe_duration'] == 0):
    logger.debug("timeframe_duration option have to be greater than 0)")
    return response_pb2.Response()
```
I'll change that

Reply:
I added a test